html/pfappserver/root/src/views/Reports/_components/TheView.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+  <b-card no-body id="card">
+    <b-card-header>
+      <h4 class="mb-0" v-html="id" />
+      <p v-if="description"
+        v-html="description" class="mt-3 mb-0" />
+    </b-card-header>
+    <div class="card-body">
+      <template v-if="isLoaded">
+        <the-search v-if="hasQuery"
+          :meta="meta"
+          :report="report"
+        >
+          <base-input-date-range v-if="hasDateRange"
+            v-model="dateRange"
+            :disabled="isLoading" />
+        </the-search>
+        <b-tabs v-if="charts.length"
+          card lazy>
```

Comment:
I think it would make sense to use the *pills* variant here since we only use icons for the tab names.

Reply:
@cgx text is optionally permitted. See asciidoc for usage.